huntsman/tests/test_camera.py

```diff
@@ -30,14 +30,16 @@
 
 # Ugly hack to access id inside fixture
 @pytest.fixture(scope='module', params=zip(params, ids), ids=ids)
-def camera(request, images_dir, camera_server):
+def camera(request, images_dir, camera_server, config_server):
```

Comment:
Redundant because the `camera_server` fixture requires the `config_server` fixture now.